src/lib/dialog/dialog-config.ts

```diff
@@ -69,5 +69,14 @@ export class MdDialogConfig {
   /** ID of the element that describes the dialog.  */
   ariaDescribedBy?: string | null = null;
 
+  /** Aria label to assign to the dialog element */
+  ariaLabel?: string | null = null;
+
+  /**
+   * Selector for an element to be focused when the dialog is opened. If omitted or the
+   * element is not found, the dialog will focus the first focusable element.
+   */
+  focusOnOpen?: string | null = null;
```

Comment:
Is there a strong reason to use a selector at all here since the focus trap already supports marking where you want the focus to start? The main thing that's missing is being able to focus the root element, which could just be a boolean.